Makefile

```
@@ -63,3 +63,131 @@ sign-release:
+		exit 1; \
+	fi
+	@echo "Uploading release assets for $(VERSION)..."
+	@for file in dist/*.zip dist/*.bundle; do \
+		if [ -f "$$file" ]; then \
+			echo "Uploading $$file..."; \
+			gh release upload "$(VERSION)" "$$file" --clobber; \
+		fi; \
+	done
+	@echo "✓ All assets uploaded successfully"
+
+.PHONY: check-commits-since-release
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)